mxenabled/connect-widget
# Initial POC

### Changes

-  It renders our first step(the search step)
- Copied over all the needed files for the search step
- Installed all required packages

### Added files/What need more attention
- `src/ConnectWidget.jsx`
- `src/redux/Store.ts`
- `withProvider.tsx` HOC which will wrap connect widget with providers
- `vite.comfig.ts'

### Testing instructions

-  Check out this branch [https://gitlab.mx.com/mx/connect-widget/-/merge_requests/207](https://gitlab.mx.com/mx/connect-widget/-/merge_requests/207) from connect-widget repo
- `npm run build` connect package and start connect widget server
- Load connect in MM and ensure the Search step is rendered
